core/templates/dev/head/pages/exploration-player-page/exploration-player-page.controller.ts

```diff
@@ -105,6 +105,8 @@ require('domain/exploration/ReadOnlyExplorationBackendApiService.ts');
 
 require('objects/objectComponentsRequiresForPlayers.ts');
 
+require('services/AutoplayedVideosService.ts');
```

Comment:
Sorry, I had an old version of develop locally. You can remove this since it is already on line 82.

Reply:
Done!